pkg/tests/scrapers/twitter_scraper_test.go

```diff
@@ -0,0 +1,158 @@
+// Package scrapers_test contains integration tests for the Twitter scraper functionality.
+//
+// Dev Notes:
+// - These tests require valid Twitter credentials set in environment variables.
+// - The tests use a temporary directory for storing cookies and other data.
+// - Make sure to run these tests in a controlled environment to avoid rate limiting.
+// - The tests cover authentication, session reuse, and basic scraping operations.
+package scrapers_test
+
+import (
+	"os"
+	"path/filepath"
+	"runtime"
+
+	"github.com/joho/godotenv"
+	"github.com/masa-finance/masa-oracle/pkg/config"
+	"github.com/masa-finance/masa-oracle/pkg/scrapers/twitter"
+	twitterscraper "github.com/masa-finance/masa-twitter-scraper"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/sirupsen/logrus"
+)
+
+var _ = Describe("Twitter Auth Function", func() {
+	var (
+		twitterUsername string
+		twitterPassword string
+		twoFACode       string
+	)
+
+	loadEnv := func() {
```

Comment:
I am a bit against loading `.env` files from the system in the integration tests - ideally we should follow the rabbit hole and try to re-factor code to not depend on a config singleton that is initialized "the first time" that "something" access to it. 

It makes it so hard to track it down where it originates and who initializes what. My suggestion here is to take a bit of time to break this down and remove dependencies from the configuration singleton in the twitter scraper code.